src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -124,6 +125,26 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
         );
     }
 
+    renderEzContent() {
+        let value = this.state.content;
+
+        if (value) {
+            value = value.replace(/&lt;/g, '<').replace(/&gt;/g, '>');
+        }
+
+        return (
+            <div className="attribute__wrapper">
+                <label className="attribute__label form-control-label">Content</label>
+                <textarea
+                    className="ezcontent__input form-control"
+                    value={value}
+                    onChange={this.updateContent.bind(this)}
```

Comment:
Please, bind the context in the `costructor` the following way: `this.updateContent = this.updateContent.bind(this);`. It will always refer to the same function instance. Everytime you use `bind()` function, you create a new copy of function.

Reply:
Done in c64f89bc970821c77d987cd0735c3a4aa5072253.